client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-add-license-notification/index.tsx

```diff
@@ -39,52 +39,60 @@ export default function SiteAddLicenseNotification() {
 		dispatch( setPurchasedLicense( license.selectedProducts.length ? license : undefined ) );
 	};
 
+	function getMessageArgs(
+		licenses: Array< { name: string; key: string; status: 'rejected' | 'fulfilled' } >
```

Comment:
I suggest giving the type here a name in client/jetpack-cloud/sections/agency-dashboard/sites-overview/types.ts. The type [here](https://github.com/Automattic/wp-calypso/blob/trunk/client/jetpack-cloud/sections/agency-dashboard/sites-overview/types.ts#L128) could be named as `ProductInfo` and then imported for use here.